include/glow/Runtime/RuntimeTypes.h

```diff
@@ -138,6 +138,16 @@ class DeviceConfig {
   void setName(llvm::StringRef name) { name_ = name; }
 };
 
+/// OpenCL Device Manager config object. This contains the information needed to
```

Comment:
Why do we have target-specific code in the target-independent part of the compiler? Do we need to add stuff like this for all of our targets?

Reply:
I moved it here so that RuntimeTypes.h had every data type needed to interface with the Runtime. I can move it back and also include OpenCLDeviceManager.h

Reply:
Why would it need to know about the specifics of one target? All of this should be hidden behind virtual calls and interfaces.

Reply:
See my comment about having a virtual method in DeviceManager to create device specific configs. We won't need to expose those backend-specific device configs if we'd use that approach or something similar.